app/views/hyku_addons/dates/_date_submitted.html.erb

```diff
@@ -1,8 +1,2 @@
-<%# see app/views/records/edit_fields/_date_submitted.html.erb %>
-<% template = f.object.model.class.to_s.underscore %>
-
-<%= render 'hyku_addons/dates/date_fields', date_type: 'submitted', template: template, f: f %>
-
-
-
-<br/><br/><br/>
+<!-- legacy - _date_submitted -->
+<%= render "hyku_addons/dates/date_fields", date_type: "submitted", template: f.object.model.class.to_s.underscore, f: f %>
```

Comment:
This is a good refactor to remove the `template` variable which is defined in this partial. But now all this partial is doing is setting the value for `date_type` and opening another partial.  The other partial _date_published and _date_accepted just set different date_types.  Can't we refactor somewhere further up the chain so `data_type` is set there and have only one partial not three?

Reply:
By default the hydra editor gem uses the `key` (underscored field name) to work out the partial to include. Because of the way that the dates are processed - by creating a JSON string of date components year, month, day, which is passed to a date processing actor and combined into a YYYY-MM-DD string - we cannot use the `_default` partial which is the last file called because a 404 is thrown. The `_default` is where I have put most of the logic for rendering the views, based on the schema definitions, but dates need more time than I currently have. 

When this work is done, then changing the way the dates are processed would be of real benefit and simply the stack - just using Rails `date(2i)` internal processing instead.